gpioioctl/basic_test.go

```diff
@@ -0,0 +1,120 @@
+package gpioioctl
+
+// Copyright 2024 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+//
+// Basic tests. More complete test is contained in the
+// periph.io/x/cmd/v3/periph-smoketest/gpiosmoketest
+// folder.
+import (
+	"testing"
+
+	"periph.io/x/conn/v3/driver/driverreg"
+	"periph.io/x/conn/v3/gpio/gpioreg"
+)
+
+var _test_line *GPIOLine
+
+func init() {
+	_, _ = driverreg.Init()
```

Comment:
Would you mind checking for errors here? Otherwise it could create hard-to-diagnose failures.

Reply:
Did you forget to push a new version? It's okay to just panic.

Reply:
Sorry, I somehow didn't commit this change.